src/security/src/main/kotlin/org/vaccineimpact/orderlyweb/security/TokenIssuer.kt

```diff
@@ -8,44 +8,65 @@ import java.security.SecureRandom
 import java.time.Duration
 import java.time.Instant
 import java.util.*
+import org.vaccineimpact.orderlyweb.db.AppConfig
 
-/**
- * We don't issue auth tokens - we rely on the main Montagu API for that.
- * However, we still need to be able to generate tokens for other things:
- * In this case, one time action tokens.
- */
-open class TokenIssuer(keyPair: KeyPair, val issuer: String)
+open class TokenIssuer(keyPair: KeyPair, val onetimeTokenIssuer: String)
 {
     val oneTimeLinkLifeSpan: Duration = Duration.ofMinutes(10)
     val signatureConfiguration = RSASignatureConfiguration(keyPair)
+
+    val appConfig = AppConfig()
+    val appName = appConfig["app.name"]
```

Comment:
I think probably sticking with `token.issuer` rather than `app.name` keeps things really clear? But in any case the issuer is a constructor argument so no need to fetch it here.